partner_daytopay/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Partner Days to Pay
+===================
+
+This module displays statistics related to the receivables and payables behavior of a partner on the Accounting tab of the partner form view.
+
+Usage
+=====
+
+* Go to the Accounting tab of a partner
+* There is 4 new fields. These fields represent the average days to pay for
+  this year (YTD) and since the first invoice (lifetime).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/94/11.0
+
```

Comment:
@max3903 Can you add in roadmap, that for payment terms with multiple payment lines, it will get the value from the date_due of the invoice, it will not threat every payment line separately, because you can have delays on first payments, but the last one to be at term, and this will not show the real delays..